virtcontainers/api.go

```diff
@@ -600,3 +601,34 @@ func ProcessListContainer(sandboxID, containerID string, options ProcessListOpti
 
 	return c.processList(options)
 }
+
+// UpdateContainer is the virtcontainers entry point to update
+// container's resources.
+func UpdateContainer(sandboxID, containerID string, resources specs.LinuxResources) error {
+	if sandboxID == "" {
+		return errNeedSandboxID
+	}
+
+	if containerID == "" {
+		return errNeedContainerID
+	}
+
+	lockFile, err := rLockSandbox(sandboxID)
+	if err != nil {
+		return err
+	}
+	defer unlockSandbox(lockFile)
+
+	p, err := fetchSandbox(sandboxID)
+	if err != nil {
+		return err
+	}
+
+	// Fetch the container.
+	c, err := p.findContainer(containerID)
```

Comment:
Please make UpdateContainer part of the VCSandbox interface and implement `UpdateContainer()` here as a wrapper of it. You can move `p.findContainer()` and `c.update()` to `sandbox.UpdateContainer()`.

Reply:
will do :)